rust/src/detect/uint.rs

```diff
@@ -45,6 +50,21 @@ pub struct DetectUintData<T> {
     pub mode: DetectUintMode,
 }
 
+pub fn detect_parse_uint_enum<T1: DetectIntType, T2: Enum<T1>>(s: &str) -> Option<DetectUintData<T1>> {
```

Comment:
I think this function needs some docs.

Reply:
indeed, let me know if I made it right for rustdoc in next PR